src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -962,33 +962,40 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
 
                 SyntaxNode placeholderSyntax;
                 uint valSafeToEscapeScope;
+                bool isSuppressed;
 
                 switch (argumentIndex)
                 {
-                    case BoundInterpolatedStringArgumentPlaceholder.InstanceParameter:
+                    case BoundInterpolatedStringArgumentPlaceholder.InstanceParameter when receiver != null:
```

Comment:
>when receiver != null [](http://example.com/codeflow?start=86&length=21)

It looks like the switch above over the same ```argumentIndex``` assumes the ```receiver``` is not null for ```case BoundInterpolatedStringArgumentPlaceholder.InstanceParameter```. Here we are guarding against null in the same situation. Is there a reason for the inconsistency?